module/x/gravity/types/validation.go

```diff
@@ -282,6 +298,33 @@ func (v *Valset) WithoutEmptyMembers() *Valset {
 	return &r
 }
 
+// Equal compares all of the valset members, additionally returning an error explaining the problem
```

Comment:
In the event we need to compare `Valset`s later I made this a dual return function